### ISSUE TITLE
Links in guides point to incorrect code locations

### ISSUE THREAD

**Issue by richnew10** [aws/amazon-chime-sdk-js#1661]
https://aws.github.io/amazon-chime-sdk-js/modules/qualitybandwidth_connectivity.html#choose-a-lower-local-video-quality

Click `chooseVideoInputQuality`. Wrong location.

**Comment by ldai1** [comment]
Good catch. Addressed the issue in this pr: https://github.com/aws/amazon-chime-sdk-js/pull/1645
